v0/destinations/posthog/data/PHGroupConfig.json

```diff
@@ -25,7 +25,7 @@
     "required": true
   },
   {
-    "destKey": "traits",
+    "destKey": "properties.$group_set",
     "sourceKeys": [
       "traits",
       "context.traits"
```

Comment:
Group traits should be taken from `traits`
Ref: https://www.rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/group/

Reply:
updated traits